### PR TITLE
docs: fix explanation of darwin auto-quit prevention

### PR DESCRIPTION
**Pull request by BenSova**
#### Description of Change
I fixed the comment above the auto-quit prevention for macOS.
Before it suggested the code quits the app, now it says the code prevents the app from quitting.

#### Release Notes

Notes: none

### CODE REVIEW

File: docs/tutorial/first-app.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ function createWindow () {
 // Some APIs can only be used after this event occurs.
 app.whenReady().then(createWindow)
 
-// Quit when all windows are closed.
+// Prevent app from auto-quitting on macOS.
```
Comment (malept) [inline docs/tutorial/first-app.md:144]:
```suggestion
// Quit when all windows are closed, except on macOS. There, it's common
// for applications and their menu bar to stay active until the user quits
// explicitly with Cmd + Q.
```

File: docs/tutorial/first-app.md [hunk 2]
```diff
@@ -141,7 +141,7 @@ function createWindow () {
...
 app.on('window-all-closed', () => {
   // On macOS it is common for applications and their menu bar
   // to stay active until the user quits explicitly with Cmd + Q
```
Comment (malept) [inline docs/tutorial/first-app.md:147]:
```suggestion
```

**Comment by erickzhao** [comment]
Thank you for the contribution! Follow-up: it would be best to update our templates in quick-start and Fiddle for consistency:

* https://github.com/electron/electron-quick-start/blob/master/main.js
* https://github.com/electron/fiddle/blob/master/src/content/main.ts

**Comment by malept** [comment]
> Follow-up: after this gets merged, it would be best to update our templates in quick-start and Fiddle for consistency

Also Electron Forge:
* [base template](https://github.com/electron-userland/electron-forge/blob/56de51a1d2ed05307c4683b40f1355863feb0627/packages/template/base/tmpl/index.js#L28-L31)
* [typescript template](https://github.com/electron-userland/electron-forge/blob/56de51a1d2ed05307c4683b40f1355863feb0627/packages/template/typescript/tmpl/index.ts#L28-L31)
* [typescript-webpack template](https://github.com/electron-userland/electron-forge/blob/56de51a1d2ed05307c4683b40f1355863feb0627/packages/template/typescript-webpack/tmpl/index.ts#L28-L31)

**Pull request merged**